config/initializers/redis.rb

```diff
@@ -0,0 +1,8 @@
+env = Figaro.env
+ttl = env.service_provider_request_ttl_hours || DEFAULT_TTL_HOURS
+REDIS_POOL = ConnectionPool.new(size: 10) do
+  Readthis::Cache.new(
+    expires_in: ttl.to_i.hours.to_i,
+    redis: { url: env.redis_throttle_url, driver: :hiredis },
```

Comment:
This always uses the secondary redis for these requests...is that intentional?

Reply:
No it was not.  It was a quick copy and paste.  I didn't realize the throttle setup a different url